app/controllers/integrations/buildkite_controller.rb

```diff
@@ -28,4 +28,9 @@ def build_param
   def message
     build_param[:message]
   end
+
+  def release_params
+    extra_params = Samson::Hooks.fire(:release_params, project, build_param)
```

Comment:
hmmm.. if i changed it to buildkite_release_params here, how to hook it up with the base controller?  Also I'm not exactly sure what you mean by adding a buildkite_controller_decorator  :(..  Can elaborate a bit? Thanks!